### PR TITLE
Removing DCAT 2 leftover

### LINKED ISSUE #1519: Check that we don't have any leftover from DCAT2
Complements https://github.com/w3c/dxwg/issues/1518

We may have leftovers from DCAT2 in running text and examples.

### PR DESCRIPTION
**Pull request by riccardoAlbertoni**
Closes #1519

### CODE REVIEW

**Comment by riccardoAlbertoni** [comment]
Other aspects we might want to consider to deal with:

- the ranges of `dcat:theme` and dcat:themeTaxonomy have been  relaxed, do we want to mirror that in fig1?

**Comment by riccardoAlbertoni** [comment]
This PR is ready for being  reviewed.  I have created a separate issue #1557  for 
> * the ranges of `dcat:theme` and dcat:themeTaxonomy have been  relaxed, do we want to mirror that in fig1?

File: dcat/index.html [hunk 1]
```diff
@@ -5318,7 +5319,7 @@ <h2>Data citation</h2>
     </p>
 
     <p>
-        In order to support data citation, this DCAT revision has added the consideration of <a href="#dereferenceable-identifiers">dereferenceable identifiers</a> and support for indicating
+        In order to support data citation, this DCAT 2 added the consideration of <a href="#dereferenceable-identifiers">dereferenceable identifiers</a> and support for indicating
```
Comment (dr-shorthair) [inline dcat/index.html]: delete `this`

File: dcat/index.html [hunk 2]
```diff
@@ -6552,7 +6553,7 @@ <h3>Dataset provenance</h3>
   <section id="examples-dataset-publication">
       <h3>Link datasets and publications</h3>
 
-      <p>Often datasets are associated with publications (scholarly articles, reports, etc.) and this version of DCAT relies on the property <a href="#Property:resource_is_referenced_by"><code>dcterms:isReferencedBy</code></a> to provide a way to link publications about a dataset to the dataset</p>
+      <p>Often datasets are associated with publications (scholarly articles, reports, etc.) and DCAT relies on the property <a href="#Property:resource_is_referenced_by"><code>dcterms:isReferencedBy</code></a> to provide a way to link publications about a dataset to the dataset</p>
```
Comment (dr-shorthair) [inline dcat/index.html]: `Datasets are often` would be better to start the sentence

**Review by dr-shorthair — CHANGES_REQUESTED** [review]
Two minor wording changes suggested

**Review by davebrowning — CHANGES_REQUESTED** [review]
I think its worthwhile making the suggested changes (especially the first one). [I can do it if you agree..?]

File: dcat/index.html (hunk 1 above)
Comment (riccardoAlbertoni) [reply dcat/index.html]:
```suggestion
        In order to support data citation, DCAT 2 added the consideration of <a href="#dereferenceable-identifiers">dereferenceable identifiers</a> and support for indicating
```

File: dcat/index.html (hunk 2 above)
Comment (riccardoAlbertoni) [inline dcat/index.html]:
```suggestion
      <p>Datasets are often associated with publications (scholarly articles, reports, etc.) and DCAT relies on the property <a href="#Property:resource_is_referenced_by"><code>dcterms:isReferencedBy</code></a> to provide a way to link publications about a dataset to the dataset</p>
```

**Comment by riccardoAlbertoni** [comment]
Changes asked by @davebrowning and @dr-shorthair have been included.

**Review by davebrowning — APPROVED** [review]

**Review by dr-shorthair — APPROVED** [review]

**Pull request merged**